10/calculator_test.go

```diff
@@ -0,0 +1,79 @@
+package calculator_test
+
+import (
+	"calculator"
+	"fmt"
+	"testing"
+)
+
+func TestSubtract(t *testing.T) {
+	testCases := []struct {
+		a, b int
+		want int
+	}{
+		{a: 1, b: 1, want: 0},
+		{a: 3, b: 2, want: 1},
+		{a: 6, b: 7, want: -1},
+	}
+	for _, testCase := range testCases {
+		got := calculator.Subtract(testCase.a, testCase.b)
+		if testCase.want != got {
+			t.Errorf("Add(%d, %d): want %d, got %d", testCase.a, testCase.b, testCase.want, got)
+		}
+	}
+}
+
+func TestDivide(t *testing.T) {
```

Comment:
It might make sense for us to move `TestDivide` so that the other three tests (which are virtually identical in structure) are grouped together.